JSDemos/Demos/Form/GroupedFields/description.md

```diff
@@ -1 +1,5 @@
-This demo illustrates how to&nbsp;organize Form items into groups and tabbed pages. For this purpose, an&nbsp;array containing &laquo;group&raquo; and &laquo;tabbed&raquo; items is&nbsp;passed to&nbsp;the items property.
\ No newline at end of file
+You can organize Form items in groups and tabs. To do this, declare items of the [Group](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/GroupItem/) and [Tabbed](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/TabbedItem/) types in the [items[]](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#items) array. Each group and tab can have their own layout and contain any other item types.
```

Comment:
Just a small suggestion - maybe say "Group and Tabbed items" instead of "items of the Group and Tabbed types".

```suggestion
You can organize Form items in groups and tabs. To do this, declare [Group](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/GroupItem/) and [Tabbed](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/TabbedItem/) items in the [items[]](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#items) array. Each group and tab can have their own layout and contain any other item types.
```